env-defaults

```diff
@@ -143,10 +143,10 @@ DATABASE_URL=
 UPLOAD_BUCKET=local-upload-bucket
 
 # SQS queue used to process work item updates
-WORK_ITEM_UPDATE_QUEUE_URL=http://localstack:4566/queue/work-item-update-queue
+WORK_ITEM_UPDATE_QUEUE_URL=http://localhost:4566/queue/work-item-update-queue
 
 # SQS queue used to process work item updates
-LARGE_WORK_ITEM_UPDATE_QUEUE_URL=http://localstack:4566/queue/large-work-item-update-queue
+LARGE_WORK_ITEM_UPDATE_QUEUE_URL=http://localhost:4566/queue/large-work-item-update-queue
```

Comment:
Please put these back so as to not break harmony-in-a-box. You can add them to your .env file.

Reply:
Thanks! Didn't realize it was for harmony in a box.